sidechain/consensus/aura/src/block_importer.rs

```diff
@@ -159,6 +180,86 @@ impl<
 	pub(crate) fn block_author_is_self(&self, block_author: &SignedSidechainBlock::Public) -> bool {
 		self.authority.public() == *block_author
 	}
+
+	fn detect_play_turn_calls(
+		&self,
+		sidechain_block: &&<SignedSidechainBlock as SignedBlock>::Block,
```

Comment:
```suggestion
		sidechain_block: &<SignedSidechainBlock as SignedBlock>::Block,
```

What's the use of a double &?